src/internal/packager/helm/chart.go

```diff
@@ -294,128 +248,106 @@ func (h *Helm) UpdateReleaseValues(ctx context.Context, updatedValues map[string
 		return nil
 	}
 
-	return fmt.Errorf("unable to find the %s helm release", h.chart.ReleaseName)
+	return fmt.Errorf("unable to find the %s helm release", chart.ReleaseName)
 }
 
-func (h *Helm) installChart(ctx context.Context, postRender *renderer) (*release.Release, error) {
+func installChart(ctx context.Context, zarfChart v1alpha1.ZarfChart, chart *chart.Chart, chartValues chartutil.Values,
```

Comment:
Nit: Not always necessary for a private fn, but it's complex enough that it should have a doccomment.

Reply:
Added doccomments to many of the functions, but I'm struggling to think of a comment for install / upgrade / rollback that would be a value add. These functions are really just setting the fields for the Helm install / upgrade / rollback function then executing it.

Reply:
Fair, that in itself can be a code smell indicating a leaky abstraction but it's not something that has to be addressed in this PR given the scope outlined in the issue.